index.src.html

```diff
@@ -286,66 +296,72 @@ <h4 id="shortlinks">Navigation</h4>
 <section>
   <h2 id="framework">Framework</h2>
 
-  <h3 id="address-space-heading">Address Space</h3>
-
-  Every IP address belongs to an IP <dfn export>address space</dfn>, which can
-  be one of three different values:
-
-  1.  <dfn for="address space" export>local</dfn>: contains the local host only.
-      In other words, addresses whose target differs for every device.
-  1.  <dfn for="address space" export>private</dfn>: contains addresses that
-      have meaning only within the current network. In other words, addresses
-      whose target differs based on network position.
-  1.  <dfn for="address space" export>public</dfn>: contains all other
-      addresses. In other words, addresses whose target is the same for all
-      devices globally on the IP network.
-
-  The contents of each [=address space=] are determined in accordance with the
-  IANA Special-Purpose Address Registries ([[IPV4-REGISTRY]] and
-  [[IPV6-REGISTRY]]). To determine the [=address space=] of a given IP address
+  <h3 id="ip-address-space-heading">IP Address Space</h3>
+
+  Every IP address belongs to an
+  <dfn export local-lt="address space">IP address space</dfn>, which can be one
```

Comment:
Nit: This is a reasonable change, but ideally you'd have broken it out into a separate PR, as it makes distinguishing the inheritance change a little more difficult.

Reply:
It's true. I was too far along with both changes when I realized I should have separated them. Thanks for taking the time to review this anyway!